### PR TITLE
Cosmos Improvments

### PR DESCRIPTION
**Pull request by Kiirx**
- [x] Make Builder installing deps faster
- [x] Reformat the entire project
- [x] Change things

### CODE REVIEW

**Comment by Kiirx** [comment]
@valentinbreiz @quajak can someone review this and merge to master if good? only INTs.cs has real changes in cosmos side. Only 1 file changed for the builder

File: .editorconfig [hunk 1]
```diff
@@ -1,30 +1,30 @@
 # EditorConfig is awesome:
 http://EditorConfig.org
 
-# top-most EditorConfig file
```
Comment (quajak) [inline .editorconfig:4]: These formatting changes look incorrect. Please revert them.

File: Docs/articles/Kernel/Network.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ VFSManager.RegisterVFS(fs);
 using(var xServer = new FtpServer(fs, "0:\\"))
 {
     /** Listen for new FTP client connections **/
-    ftpSever.Listen();
+    FtpServer.Listen();
```
Comment (quajak) [inline Docs/articles/Kernel/Network.md:95]: Should this not be `xServer`?

File: source/Cosmos.Common/BinaryHelper.cs [hunk 3]
```diff
@@ -1,133 +1,138 @@
 ﻿using System;
 
-namespace Cosmos.Common
+namespace Cosmos.Common;
```
Comment (quajak) [inline source/Cosmos.Common/BinaryHelper.cs:3]: I dont think we should change the namespace format everywhere.

**Review by quajak — CHANGES_REQUESTED** [review]
See my intial requests. The changes to spacing due to the namespace format change makes it impossible to review the rest of the changes. Can you please revert it? In future, please discuss changes to formatting/style guide here on github or on the discord before applying them.

File: source/Cosmos.Common/BinaryHelper.cs (hunk 3 above)
Comment (Kiirx) [reply source/Cosmos.Common/BinaryHelper.cs:3]: Its the Auto-formatter from JetBrains Rider

File: .editorconfig (hunk 1 above)
Comment (Kiirx) [reply .editorconfig:4]: Why

File: Docs/articles/Kernel/Network.md (hunk 2 above)
Comment (Kiirx) [reply Docs/articles/Kernel/Network.md:95]: It never was

**Comment by Lorenmod** [comment]
Kindly Live chat cosmos Agent on live chat support https://cosmossnetwork.com if you have issue or concerns

**Comment by Kiirx** [comment]
Can you revert these reformat changes I made?

**Comment by quajak** [comment]
I can't easily do that. Most likely the best way is if you determine in which commit you added them and revert that commit and reapply any changes you want to keep from that commit.

**Comment by Kiirx** [comment]
Can you revert these reformat changes I made? Alright

**Review by quajak — APPROVED** [review]

**Pull request merged**